Cleanup the default blog list and blog post styles

raghukamath opened the pull request:
- Remove hard coded inline styles from the blog post listing.
- Add necessary spacing and styling to the list page and blog post page.
- remove the Clipping of cover image in the blog list page

raghu-kamath committed:
make changes in the blog templates

raghu-kamath committed:
Merge branch 'develop' of https://github.com/frappe/frappe into blog-refresh

netchampfaris commented on frappe/website/doctype/blog_post/templates/blog_post_row.html:
```diff
@@ -4,13 +4,13 @@
-			<p style="height: 200px; overflow: hidden;">
-				<img src="{{ post.cover_image }}" alt="{{post.title}} - Cover Image" style="width: 60%;"/ style='blog-cover-image'>
+			<p>
+				<img class="blopost-cover-img" src="{{ post.cover_image }}" alt="{{post.title}} - Cover Image">
```
typo

netchampfaris commented on frappe/templates/includes/blog/blog.html:
```diff
@@ -5,25 +5,6 @@
 {% block hero %}{% endblock %}
 
 {% block page_content %}
-<style>
```
Put these styles in website.less

raghukamath replied:
Thanks for the review I added the styles to website.less and also tweak the styles a bit for adding spacing around avatar and blog posts

raghu-kamath committed:
Add the inline styles to website.less
I also tweaked the design a bit, added h1 for header and increased the spacing a bit.

raghukamath commented:
After the changes the vanilla blog page looks like this 
![screenshot-2018-1-24 blog](https://user-images.githubusercontent.com/9101092/35341541-eaaf7c1c-014b-11e8-9200-e38f84111727.jpg)

raghu-kamath committed:
Remove inline style and add css to less file

also added some spacing to make the blogpost look good by default.

raghu-kamath committed:
make the headings bold
